Sources/FluentSQL/SQL+QuerySupporting.swift

```diff
@@ -157,6 +157,23 @@ extension QuerySupporting where
     QueryKey: SQLSelectExpression,
     QueryKey.Expression == QueryKey.Expression.Function.Argument.Expression
 {
+    /// See `QuerySupporting`.
+    public static func queryAggregate<D>(_ name: QueryAggregate, _ fields: [QueryKey], default: D) -> QueryKey
+        where D: Decodable
+    {
+        let args: [QueryKey.Expression.Function.Argument] = fields.compactMap { expr in
+            if expr.isAll {
+                return .all
+            } else if let (expr, _) = expr.expression {
+                return .expression(expr)
+            } else {
+                return nil
+            }
+        }
+        
+        return .expression(.coalesce(.function(.function(name, args)), .literal(.numeric(String(describing: `default`)))), alias: .identifier("fluentAggregate"))
```

Comment:
I think this could risk SQL injection. We need to bind any values the user could input.